Luiz-lgontijo/Homework-1/BONUS-madlibs/js/app.js

```diff
@@ -0,0 +1,25 @@
+const startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+const startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+let string;
+let fav = [];
+
+function displayStartup(){
+	var random1 = Math.floor((Math.random() * startupX.length));
```

Comment:
The new version of JavaScript, ES6, introduces let in favor of var; this new type of variable keeps scope, which works great in loops and conditionals.